migrator/migrations/1_create_users_table.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE public.users
+(
+    id uuid NOT NULL,
+    user_name text NOT NULL,
```

Comment:
Text used for long strings VARCHAR is much better

Reply:
What about `username`, `first_name`, `last_name`, etc?
For example: 
```
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer lobortis aliquam elit, ut molestie mi. Sed semper orci ut arcu accumsan lacinia. Vivamus mollis tellus molestie nulla iaculis luctus. Morbi at suscipit risus. Ut aliquam ultrices commodo. Quisque hendrerit ornare luctus. Fusce scelerisque lectus ac tellus finibus sagittis. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Sed tincidunt lacus quis lectus luctus auctor.
```

Reply:
The longest name used by anyone is Adolph Blaine Charles David Earl Frederick Gerald Hubert Irvin John Kenneth Lloyd Martin Nero Oliver Paul Quincy Randolph Sherman Thomas Uncas Victor William Xerxes Yancy Wolfeschlegelsteinhausenbergerdorff, Senior, who was born at Bergedorf, near Hamburg, Germany, on 29 Feb. 1904. On printed forms he uses only his eighth and second Christian names and the first 35 letters of his surname. The full version of the name of 590 letters appeared in the 12th edition of The Guinness Book of Records. He now lives in Philadelphia, Pennsylvania, U.S.A., and has shortened his surname to Mr. Wolfe + 585, Senior.  http://hoaxes.org/weblog/comments/worlds_longest_surname

Reply:
Do you think i should change to varchar?

Reply:
fix it

Reply:
> The longest name used by anyone is Adolph Blaine Charles David Earl Frederick Gerald Hubert Irvin John Kenneth Lloyd Martin Nero Oliver Paul Quincy Randolph Sherman Thomas Uncas Victor William Xerxes Yancy Wolfeschlegelsteinhausenbergerdorff, Senior, who was born at Bergedorf, near Hamburg, Germany, on 29 Feb. 1904. On printed forms he uses only his eighth and second Christian names and the first 35 letters of his surname. The full version of the name of 590 letters appeared in the 12th edition of The Guinness Book of Records. He now lives in Philadelphia, Pennsylvania, U.S.A., and has shortened his surname to Mr. Wolfe + 585, Senior. http://hoaxes.org/weblog/comments/worlds_longest_surname

Good point. And misunderstanding. Username is a synonym of login in this scope, and it shouldn't be long. For stuff you mentioned user can use First name or lat name fields.